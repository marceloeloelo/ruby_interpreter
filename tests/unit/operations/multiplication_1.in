1*1
puts(1*1)

1*2.3
puts(1*2.3)

1.2*3
puts(1.2*3)

1.2*3.4
puts(1.2*3.4)

1.2 *   3.4
puts(1.2 *   3.4)

puts(1.5*0.9)
