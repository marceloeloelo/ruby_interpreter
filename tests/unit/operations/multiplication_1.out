1
2.3
3.6
4.08
4.08
1.35
